drivers/bch/bchlib_write.c

```diff
@@ -56,13 +56,16 @@ ssize_t bchlib_write(FAR void *handle, FAR const char *buffer, off_t offset,
         size_t len)
 {
   FAR struct bchlib_s *bch = (FAR struct bchlib_s *)handle;
-  size_t   nsectors;
   size_t   sector;
   uint16_t sectoffset;
   size_t   nbytes;
   size_t   byteswritten;
   int      ret;
 
+#ifndef CONFIG_BCH_FORCE_INDIRECT
+  size_t   nsectors;
```

Comment:
Revert this, both direct  / indirect need nsectors

Reply:
I'm not using nsectors in case of CONFIG_BCH_FORCE_INDIRECT

Reply:
Oh, I missed that!

Please move this to original position though!